cmd/youtubeuploader/main.go

```diff
@@ -104,12 +104,19 @@ func main() {
 		RecordingDate:     recordingDate,
 	}
 
-	config.Logger = utils.NewLogger(*debug)
-
-	config.Logger.Debugf("Youtubeuploader version: %s\n", appVersion)
+	// setup logging
+	programLevel := new(slog.LevelVar) // Info by default
+	so := &slog.HandlerOptions{Level: programLevel}
+	if *debug {
+		//so.AddSource = true
+		programLevel.Set(slog.LevelDebug)
+	}
+	logger := slog.New(slog.NewTextHandler(os.Stderr, so))
+	slog.SetDefault(logger)
 
+	slog.Debug("youtubeuploader version", "version", appVersion)
 	if config.ShowAppVersion {
-		fmt.Printf("Youtubeuploader version: %s\n", appVersion)
+		// exit immediatly after showing version
```

Comment:
Now the version command shows nothing, is this intended?

Reply:
Thanks for spotting that! Fixed in master.